packages/browser/src/plugins/segmentio/batched-dispatcher.ts

```diff
@@ -92,7 +93,7 @@ export default function batch(apiHost: string, config?: BatchingConfig) {
     }, timeout)
   }
 
-  window.addEventListener('beforeunload', () => {
+  onPageLeave(() => {
     pageUnloaded = true
```

Comment:
Is `pageUnloaded` still needed now that `onPageLeave()` has its own check?

Reply:
I don't think anything changed in that regard, the variable really should be called "keepAlive"